peekingduck/pipeline/nodes/model/yolact_edgev1/yolact_edge_model.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2022 AI Singapore
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""YolactEdge models"""
+
+import logging
+from typing import Any, Dict, List, Tuple
+import numpy as np
+
+from peekingduck.pipeline.nodes.base import (
+    ThresholdCheckerMixin,
+    WeightsDownloaderMixin,
+)
+from peekingduck.pipeline.nodes.model.yolact_edgev1.yolact_edge_files.detector import (
+    Detector,
+)
+
+
+class YolactEdgeModel(ThresholdCheckerMixin, WeightsDownloaderMixin):
+    """YolactEdge model with ResNet 101 FPN backbone"""
+
+    def __init__(self, config: Dict[str, Any]) -> None:
+        self.config = config
+        self.logger = logging.getLogger(__name__)
+
+        model_dir = self.download_weights()
+
+        with open(model_dir / self.weights["classes_file"]) as infile:
+            class_names = [line.strip() for line in infile.readlines()]
+
+        self.detect_ids = self.config["detect"]  # change "detect_ids" to "detect"
```

Comment:
This comment was relevant to YOLOX as its original code read `self.detect_ids = self.config["detect_ids"]`, but it doesn't look applicable here.